app/templates/users/browse_users.html

```diff
@@ -17,29 +17,21 @@ <h3 class="title">Search by Username</h3>
         <tbody>
             {% for profile in users %}
             <tr>
-                <th scope="row">{{ profile }}</th>
+                <th scope="row">
+                    <a href="{% url 'user_profile' profile.user.id %}"
+                        class="text-decoration-none text-dark">
+                    {{ profile }}
+                </th>
     
                 <td>{{ profile.user.first_name }} {{ profile.user.last_name }}</td>
                 <td> Upcoming: {{ profile.upcomingEvent|default:'None' }}</td>
                 <td>Past Events: {{ profile.pastEvent|default:'None' }}</td>
                 <td>{{ profile.bio }}</td>
                 <td>
                     <button type="button" class="addBtn">Add</button>
```

Comment:
This add button should also be wired up - it's fine if you don't want to do it in this PR though